### PR TITLE
Add See also links /7

### LINKED ISSUE #30864: Add useful See also links
### Description

[Enter feedback here]

### Page URL

https://learn.microsoft.com/en-us/aspnet/core/release-notes/aspnetcore-8.0?view=aspnetcore-8.0

### Content source URL

https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/release-notes/aspnetcore-8.0.md

### Document ID

f30dbb9e-d64d-a925-01ca-00d9d035e6c1

### Article author

Rick-Anderson

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #30864

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/fundamentals/aot/native-aot-tutorial.md](https://github.com/dotnet/AspNetCore.Docs/blob/0604f9b7042ddd589afb5b9b5dbae0fb488a7efb/aspnetcore/fundamentals/aot/native-aot-tutorial.md) | [Tutorial: Publish an ASP.NET Core app using native AOT](https://review.learn.microsoft.com/en-us/aspnet/core/fundamentals/aot/native-aot-tutorial?branch=pr-en-us-30865) |
| [aspnetcore/fundamentals/minimal-apis.md](https://github.com/dotnet/AspNetCore.Docs/blob/0604f9b7042ddd589afb5b9b5dbae0fb488a7efb/aspnetcore/fundamentals/minimal-apis.md) | [Differences between minimal APIs and APIs with controllers](https://review.learn.microsoft.com/en-us/aspnet/core/fundamentals/minimal-apis?branch=pr-en-us-30865) |
| [aspnetcore/fundamentals/native-aot.md](https://github.com/dotnet/AspNetCore.Docs/blob/0604f9b7042ddd589afb5b9b5dbae0fb488a7efb/aspnetcore/fundamentals/native-aot.md) | [ASP.NET Core support for native AOT](https://review.learn.microsoft.com/en-us/aspnet/core/fundamentals/native-aot?branch=pr-en-us-30865) |
| [aspnetcore/host-and-deploy/docker/building-net-docker-images.md](https://github.com/dotnet/AspNetCore.Docs/blob/0604f9b7042ddd589afb5b9b5dbae0fb488a7efb/aspnetcore/host-and-deploy/docker/building-net-docker-images.md) | [Docker images for ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/host-and-deploy/docker/building-net-docker-images?branch=pr-en-us-30865) |
| [aspnetcore/host-and-deploy/docker/visual-studio-tools-for-docker.md](https://github.com/dotnet/AspNetCore.Docs/blob/0604f9b7042ddd589afb5b9b5dbae0fb488a7efb/aspnetcore/host-and-deploy/docker/visual-studio-tools-for-docker.md) | [aspnetcore/host-and-deploy/docker/visual-studio-tools-for-docker](https://review.learn.microsoft.com/en-us/aspnet/core/host-and-deploy/docker/visual-studio-tools-for-docker?branch=pr-en-us-30865) |
| [aspnetcore/mvc/controllers/dependency-injection.md](https://github.com/dotnet/AspNetCore.Docs/blob/0604f9b7042ddd589afb5b9b5dbae0fb488a7efb/aspnetcore/mvc/controllers/dependency-injection.md) | [Dependency injection into controllers in ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/mvc/controllers/dependency-injection?branch=pr-en-us-30865) |

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@andrewlock please review

File: aspnetcore/fundamentals/aot/native-aot-tutorial.md [hunk 1]
```diff
@@ -161,3 +162,8 @@ info: Microsoft.Hosting.Lifetime[0]
 
 * <xref:fundamentals/native-aot>
 * [Native AOT deployment](/dotnet/core/deploying/native-aot/)
+* [Using the configuration binder source generator](https://andrewlock.net/exploring-the-dotnet-8-preview-using-the-new-configuration-binder-source-generator/)
+* [The minimal API AOT compilation template](https://andrewlock.net/exploring-the-dotnet-8-preview-the-minimal-api-aot-template/)
+* [Comparing `WebApplication.CreateBuilder` to `CreateSlimBuilder`](https://andrewlock.net/exploring-the-dotnet-8-preview-comparing-webapplication-createbuilder-to-createslimbuilder/)
+* [Exploring the new minimal API source generator](https://andrewlock.net/exploring-the-dotnet-8-preview-the-minimal-api-source-generator/)
```
Comment (andrewlock) [inline aspnetcore/fundamentals/aot/native-aot-tutorial.md:168]:
```suggestion
* [Exploring the new minimal API source generator](https://andrewlock.net/exploring-the-dotnet-8-preview-exploring-the-new-minimal-api-source-generator/)
```

File: aspnetcore/fundamentals/aot/native-aot-tutorial.md [hunk 2]
```diff
@@ -161,3 +162,8 @@ info: Microsoft.Hosting.Lifetime[0]
...
+* [Replacing method calls with Interceptors](https://andrewlock.net/exploring-the-dotnet-8-preview-replacing-method-calls-with-interceptors/)
```
Comment (andrewlock) [inline aspnetcore/fundamentals/aot/native-aot-tutorial.md:169]:
```suggestion
* [Replacing method calls with Interceptors](https://andrewlock.net/exploring-the-dotnet-8-preview-changing-method-calls-with-interceptors/)
```

File: aspnetcore/fundamentals/native-aot.md [hunk 3]
```diff
@@ -258,3 +261,8 @@ See [this GitHub issue](https://github.com/dotnet/core/issues/8288) to report or
 
 * <xref:fundamentals/native-aot-tutorial>
 * [Native AOT deployment](/dotnet/core/deploying/native-aot/)
+* [Using the configuration binder source generator](https://andrewlock.net/exploring-the-dotnet-8-preview-using-the-new-configuration-binder-source-generator/)
+* [The minimal API AOT compilation template](https://andrewlock.net/exploring-the-dotnet-8-preview-the-minimal-api-aot-template/)
+* [Comparing `WebApplication.CreateBuilder` to `CreateSlimBuilder`](https://andrewlock.net/exploring-the-dotnet-8-preview-comparing-webapplication-createbuilder-to-createslimbuilder/)
```
Comment (andrewlock) [inline aspnetcore/fundamentals/native-aot.md:266]:
```suggestion
* [Comparing `WebApplication.CreateBuilder` to `CreateSlimBuilder`](https://andrewlock.net/exploring-the-dotnet-8-preview-comparing-createbuilder-to-the-new-createslimbuilder-method/)
```

File: aspnetcore/fundamentals/native-aot.md [hunk 4]
```diff
@@ -258,3 +261,8 @@ See [this GitHub issue](https://github.com/dotnet/core/issues/8288) to report or
...
+* [Exploring the new minimal API source generator](https://andrewlock.net/exploring-the-dotnet-8-preview-the-minimal-api-source-generator/)
```
Comment (andrewlock) [inline aspnetcore/fundamentals/native-aot.md:267]:
```suggestion
* [Exploring the new minimal API source generator](https://andrewlock.net/exploring-the-dotnet-8-preview-exploring-the-new-minimal-api-source-generator/)
```

File: aspnetcore/fundamentals/native-aot.md [hunk 5]
```diff
@@ -258,3 +261,8 @@ See [this GitHub issue](https://github.com/dotnet/core/issues/8288) to report or
...
+* [Replacing method calls with Interceptors](https://andrewlock.net/exploring-the-dotnet-8-preview-replacing-method-calls-with-interceptors/)
```
Comment (andrewlock) [inline aspnetcore/fundamentals/native-aot.md:268]:
```suggestion
* [Replacing method calls with Interceptors](https://andrewlock.net/exploring-the-dotnet-8-preview-changing-method-calls-with-interceptors/)
```

File: aspnetcore/host-and-deploy/docker/building-net-docker-images.md [hunk 6]
```diff
@@ -178,6 +178,7 @@ In the preceding *Dockerfile*, the `*.csproj` files are copied and restored as d
 * [Debugging with Visual Studio Code](https://code.visualstudio.com/docs/nodejs/debugging-recipes#_debug-nodejs-in-docker-containers)
 * [GC using Docker and small containers](xref:performance/memory#sc)
 * [System.IO.IOException: The configured user limit (128) on the number of inotify instances has been reached](xref:host-and-deploy/docker/index#d128)
+* [Updates to Docker images](https://andrewlock.net/exploring-the-dotnet-8-preview-updates-to-docker-images-in-dotnet-8/)
```
Comment (andrewlock) [inline aspnetcore/host-and-deploy/docker/building-net-docker-images.md:181]:
```suggestion
* [Updates to Docker images](https://andrewlock.net/exploring-the-dotnet-8-preview-updates-to-docker-images-in-dotnet-8/)
```

File: aspnetcore/host-and-deploy/docker/visual-studio-tools-for-docker.md [hunk 7]
```diff
@@ -238,3 +238,4 @@ There may be an expectation for the production or release image to be smaller in
 * [Visual Studio Container Tools GitHub repository](https://github.com/Microsoft/DockerTools)
 * [GC using Docker and small containers](xref:performance/memory#sc)
 * [System.IO.IOException: The configured user limit (128) on the number of inotify instances has been reached](xref:host-and-deploy/docker/index#d128)
+* [Updates to Docker images](https://andrewlock.net/exploring-the-dotnet-8-preview-updates-to-docker-images-in-dotnet-8/)
```
Comment (andrewlock) [inline aspnetcore/host-and-deploy/docker/visual-studio-tools-for-docker.md:241]:
```suggestion
* [Updates to Docker images](https://andrewlock.net/exploring-the-dotnet-8-preview-updates-to-docker-images-in-dotnet-8/)
```

**Review by andrewlock — COMMENTED** [review]
I've updated a bunch of the links, but for some of them I wasn't sure which post you were aiming for! 😅

File: aspnetcore/fundamentals/minimal-apis.md [hunk 8]
```diff
@@ -239,6 +232,16 @@ Moved to uid: tutorials/min-web-api
 * <xref:fundamentals/minimal-apis/handle-errors>
 * <xref:fundamentals/minimal-apis/security>
 * <xref:fundamentals/minimal-apis/test-min-api>
+* [Short-circuit routing](https://andrewlock.net/exploring-the-dotnet-8-preview-short-circuit-routing/)
+* [Identity API endpoints](https://andrewlock.net/exploring-the-dotnet-8-preview-introducing-the-identity-api-endpoints/)
+* [Keyed service dependency injection container support](https://andrewlock.net/exploring-the-dotnet-8-preview-keyed-services-dependency-injection-support/)
+* [A look behind the scenes of minimal API endpoints](https://andrewlock.net/a-first-look-behind-the-scenes-of-minimal-api-endpoints/)
+* [A look behind the scenes of minimal API routing](https://andrewlock.net/a-look-behind-the-scenes-of-minimal-api-routing/)
```
[the remark this answers is not in the document]
Comment (Rick-Anderson) [reply aspnetcore/fundamentals/minimal-apis.md:239]: I'll check all these. Code pilot, the chatGPT AI bot suggested those links so I accepted the suggestion without testing. It sometimes makes stuff up.

**Pull request merged**